modules/checklist/steps/setup-header.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Elementor\Modules\Checklist\Steps;
+
+use Elementor\Core\Isolation\Wordpress_Adapter_Interface;
+use Elementor\Modules\Checklist\Module as Checklist_Module;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Setup_Header extends Step_Base {
+	const STEP_ID = 'setup_header';
+
+	public function __construct( ...$args ) {
+		call_user_func( [ parent::class, __FUNCTION__ ], ...$args );
+
+		$this->set_is_locked( true );
+	}
+
+	public function get_id() : string {
+		return self::STEP_ID;
+	}
+
+	public function is_absolute_completed() : bool {
+		$args = [
+			'post_type' => 'elementor_library',
+			'meta_query' => [
+				'relation' => 'AND',
+				[
+					'key' => '_elementor_template_type',
+					'value' => 'header',
+					'compare' => '=',
+				],
+				[
+					'key' => '_elementor_conditions',
+				],
+			],
+			'posts_per_page' => 1,
+		];
```

Comment:
Please add the following args:

```
'fields' => 'ids',
'no_found_rows' => true,
'update_post_term_cache' => false,
'update_post_meta_cache' => false,
```
(for performance)

Reply:
Thanks!